api/app/Admin.js

```diff
@@ -61,4 +61,32 @@ module.exports = function() {
             console.log(e);
         }
     });
+
+    //MongoDb Native Access
+    global.app.post('/admin/dbaccess/enable/:appId',function(req, res) {
+        console.log("++++ MongoDb Native Access ++++++");
+        try {    
+            if (global.keys.secureKey === req.body.secureKey) {
+                var username = Math.random().toString(36).substring(7)
```

Comment:
This should not be in API layer, but should be in services layer. Also check util helper to generate a random string.